src/components/inputs/preview.vue

```diff
@@ -96,31 +97,54 @@
                                         }   
                                     });
                                 </script>
-                                </textarea>
-                            </div>
-                        </div>
-                    </div>               
-                </div>
-            </div>        
+                                </textarea
+                >
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+
+    <div class="row">
+      <div class="col-12">
+        <div class="card">
+          <div class="card-header">KmInputLString</div>
+          <div class="card-body">
+            <div class="row">
+              <div class="col-6">
+                <km-input-lstring
+                  v-model="kmInputLStringModel"
+                  :locales="locales"
+                >
+                </km-input-lstring>
+              </div>
+              <div class="col-4">
+                <strong>Km Input L sample code</strong>
+              </div>
+            </div>
```

Comment:
can you use https://github.com/scbd/scbd-common/pull/29/commits/f736d2e13f66ec1a95e644efae680d46ab813593

Reply:
used `code` tag from aforementioned commit to add code of `km-input-lstring.vue` file